### PR TITLE
Update CONTRIBUTING doc

### PR DESCRIPTION
**Pull request by zyfdegh**
Add more helpful info for new users in CONTRIBUTING.md to avoid confusion and errors

### CODE REVIEW

**Comment by zyfdegh** [comment]
Please merge with `--squash`

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,6 +28,17 @@ cp .env_example .env
 vim .env
 ```
 
+For example if your [api](https://github.com/MysteriumNetwork/api) is listening
+on `your.hostname.com:8001`, then the content of the `.env` file should look like this
+
+```
+MYSTERIUM_API_URL=http://your.hostname.com:8001/v1
```
Comment (Waldz) [inline CONTRIBUTING.md]: No need for external address, could be `http://localhost:8001/v1`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -28,6 +28,17 @@ cp .env_example .env
 vim .env
 ```
 
+For example if your [api](https://github.com/MysteriumNetwork/api) is listening
```
Comment (Waldz) [inline CONTRIBUTING.md]: Idea is to `cp .env_example .env` and edit file.
Could You add these recommendations to `.env_example` file

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -43,6 +54,41 @@ bin/client_build
 bin/client_run
 ```
 
+Running client in interative cli
```
Comment (Waldz) [inline CONTRIBUTING.md]: +100 kudos

**Pull request merged**